addons/io_scene_gltf2/io/exp/gltf2_io_draco_compression_extension.py

```diff
@@ -85,6 +85,10 @@ def __encode_primitive(primitive, dll, export_settings):
     attributes = primitive.attributes
     indices = primitive.indices
 
+    # Only do TRIANGLES primitives
+    if primitive.mode not in [None, 4]:
+        return
```

Comment:
Despite this check, I'm still getting an error when I enable Draco export. Here's a test case:

[Uploading LooseTest.blend.zip…]()

Reply:
I get that anytime I try to export with "Compresssion" on, even with master.

Reply:
Hm, I'm definitely willing to believe it's a version mismatch or some detail of the native Draco integration with local Blender, if it's working for someone else. I typically don't do compression until after exporting, with an offline tool.

That said ... the sample above has no triangles at all, so ideally it wouldn't be trying to create a Draco encoder.

This doesn't seem like a blocking issue, then, in any case.